buildSrc/src/main/kotlin/Extensions.kt

```diff
@@ -54,49 +57,73 @@ private inline fun <T> List<T>.sumByIndexed(selector: (Int, T) -> Int): Int {
     return sum
 }
 
-private fun DependencyHandler.testImplementation(dependencyNotation: Any): Dependency? {
-    return add("testImplementation", dependencyNotation)
-}
+private fun DependencyHandler.testImplementation(vararg names: Any): Array<Dependency?> =
```

Comment:
This is personal preference, but I'd rather leave this part as is (even if the `testImplementation` and `androidTestImplementation` statements are a bit repetitive).

Reply:
Reverted.